client-cli/src/command/transaction_command.rs

```diff
@@ -1009,13 +1024,97 @@ fn ask_transfer_address() -> Result<ExtendedAddr> {
     Ok(address)
 }
 
-fn ask_node_metadata() -> Result<CouncilNode> {
+fn keypackage_info(keypackage: &KeyPackage) -> Result<String> {
+    let mut credential: Vec<u8> = vec![];
+    keypackage.payload.credential.encode(&mut credential);
+    let extensions = keypackage
+        .payload
+        .extensions
+        .iter()
+        .map(|e| {
+            let mut extension_entry: Vec<u8> = vec![];
+            e.encode(&mut extension_entry);
+            base64::encode(&extension_entry)
+        })
+        .collect::<Vec<_>>();
+    let credential = base64::encode(&credential);
+    let l = credential.len();
+    let credential_info = if l > 64 {
+        format!("{}........{}", &credential[0..32], &credential[l - 32..])
+    } else {
+        credential
+    };
+    let life_time = keypackage
+        .payload
+        .find_extension::<LifeTimeExt>()
+        .chain(|| {
+            (
+                ErrorKind::IllegalInput,
+                "can not find extention from keypackage",
+            )
+        })?;
+    let extension_begin = Local.timestamp(life_time.not_before as i64, 0);
```

Comment:
add some comment that this is `local` time

Reply:
also genesis time is utc time, how about specify utc/local time together

Reply:
the timezone will be included in the result,  like: `2020-06-23T10:17:55.177825+08:00`, UTC time is not friendly for humans.